test/completion.js

```diff
@@ -121,13 +121,20 @@ describe('Completion', function () {
       // SKipping on CI right now, otherwise might introduce an `npm install
       // generator-dummy` as a pretest script
 
-      it.skip('Returns the list of user installed generators as completion candidates', function (done) {
+      before(function(done) {
```

Comment:
Hey, I answered you on a new review comment. I think it'd be better to not do it.

We did that in yeoman-environment, and it's really painful. Tests became slow and are constantly flaking.